go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	github.com/go-openapi/swag v0.19.8 // indirect
 	github.com/go-sql-driver/mysql v1.4.1
 	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.4.0
+	github.com/golang/protobuf v1.3.5
```

Comment:
I think the changes to `go.mod` and `go.sum` were unintentional. Could you revert them back by doing `git checkout master go.mod go.sum`?